<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelComplexityDialog</class>
 <widget class="QDialog" name="LevelComplexityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сложность игры</string>
  </property>
  <layout class="QGridLayout" name="main_layout_">
   <item row="0" column="2">
    <widget class="QSpinBox" name="spin_box_number_">
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_number_">
     <property name="text">
      <string>Количество загаданных цифр</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="button_box_">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="extra_box_">
     <property name="title">
      <string>Дополнительные опции</string>
     </property>
     <layout class="QGridLayout" name="extra_layout_">
      <item row="2" column="0">
       <widget class="QLabel" name="label_timer_">
        <property name="text">
         <string>Ограничение по времени</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTimeEdit" name="timer_"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_step_">
        <property name="text">
         <string>Ограничение на количество шагов</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spin_box_limit_steps_"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="button_extra_options_">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_extra_options_">
     <property name="text">
      <string>Включить дополнительные опции</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box_</sender>
   <signal>accepted()</signal>
   <receiver>LevelComplexityDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box_</sender>
   <signal>rejected()</signal>
   <receiver>LevelComplexityDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
